src/main.cpp

```diff
@@ -2417,6 +2685,33 @@ void vcRenderWindow(vcState *pProgramState)
       }
     }
 
+	if (pProgramState->settings.window.windowsOpen[vcDocks_History])
+	{
+		vcRender_SwitchContext(pProgramState, vcContext_History);
+		if (!pProgramState->settings.window.presentationMode)
+		{
+			if (ImGui::Begin(udTempStr("%s###historyDock", vcString::Get("historyTitle")), &pProgramState->settings.window.windowsOpen[vcDocks_History]))
+				vcRenderHistoryWindow(pProgramState);
+			vcChangeTab(pProgramState, vcDocks_History);
+
+			ImGui::End();
+		}
+		else
+		{
+			ImGui::SetNextWindowSize(size);
+			ImGui::PushStyleVar(ImGuiStyleVar_WindowPadding, ImVec2(2, 2));
+			ImGui::SetNextWindowPos(ImVec2(0, 0));
+
+			bool sceneWindow = ImGui::Begin(udTempStr("%s###scenePresentation", vcString::Get("historyTitle")), &pProgramState->settings.window.windowsOpen[vcDocks_History]);
```

Comment:
This will probably collide with the existing scenePresentation window if they can ever exist simultaneously, the part after ### is used as a unique ID if it's present...